src/gui/presets.cpp

```diff
@@ -1014,6 +1103,19 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Cave 68000", {
+      DIV_SYSTEM_YMZ280B, 64, 0, 0,
+      0
+    }
+  ));
+        cat.systems.push_back(FurnaceGUISysDef(
+    "SNK Triple Z80", {
+      DIV_SYSTEM_Y8950, 64, 0, 0,
+      DIV_SYSTEM_OPL, 64, 0, 0,
```

Comment:
OPL2 is also used instead OPL1 but only for Chopper I.

Reply:
not going to add this until m6295 gets implemented.

Reply:
It __**is**__ implemented, albeit partially.

Reply:
done